src/utils/apiTypes.ts

```diff
@@ -16,5 +18,6 @@ export type GetQuestionsResponse = {
 
 export type Unit = {
   id: number,
-  name: string
+  name: string,
+  type: UnitType
```

Comment:
Maybe make type UnitType | undefine since it is not needed when we return all units.

Reply:
1) Can you help me understand why we don't need type when return all units? In my opinion, if the Unit Object is passing around in the code, it is more convivence to have type within the object, then you don't have to look it up every time thru a lookup table.
2) I prefer to keep most of them strongly typed to avoid runtime error in the future.

Reply:
The contract of get /units endpoint's response is 
`{
units: [{
id: number,
name: string
}]
}`
So we don't need type in getAllUnit().
I think it will be fine to make `type: Unit` to `type: UnitType | undefine` since it just make `type` optional.

Reply:
Ok, I will remove type

Reply:
No need to remove it, just make it optional with `type: UnitType | undefine` maybe also rename `type` to `unitType` to make it more descriptive. Thanks!